lib/collection/src/shards/local_shard.rs

```diff
@@ -50,6 +51,24 @@ pub struct LocalShard {
 
 /// Shard holds information about segments and WAL.
 impl LocalShard {
+    /// Clear local shard related data.
+    ///
+    /// Do NOT remove config file.
```

Comment:
Can you elaborate a bit more - why not to remove config

Reply:
because it is now part of the replica set, it have replicate state config in there. After migration we can get rid of other configs